src/ChronosInterface.php

```diff
@@ -539,6 +539,39 @@ public function subYear($value = 1);
      */
     public function subYears($value);
 
+    /**
+     * Add years with overflowing to the instance. Positive $value
+     * travels forward while negative $value travels into the past.
+     *
+     * @param int $value The number of years to add.
+     * @return static
+     */
+    public function addYearsWithOverflow($value);
+
+    /**
+     * Add a year with overflow to the instance
+     *
```

Comment:
This is missing an explanation of overflow and needs the same example as ``addYears()`` but with the new result.

If the new date does not exist, the date overflows into the next month.